routes/movie.routes.js

```diff
@@ -1,9 +1,9 @@
  const movieController = require('../controllers/movie.controller');
  const {authJwt} = require('../middlewares')
-
+ const {movieValid}=require("../middlewares/index")
  module.exports = (app)=>{
-     app.post("/mba/api/v1/movies", [authJwt.verifyToken, authJwt.isAdmin], movieController.createNewMovie);
-     app.put("/mba/api/v1/movies/:id", [authJwt.verifyToken, authJwt.isAdmin], movieController.editMovie)
+     app.post("/mba/api/v1/movies", [authJwt.verifyToken,movieValid.movieValidation, authJwt.isAdmin], movieController.createNewMovie);
+     app.put("/mba/api/v1/movies/:id", [authJwt.verifyToken,movieValid.movieValidation, authJwt.isAdmin], movieController.editMovie)
```

Comment:
In the case of PUT entire request body is not needed. Now after this MW, put will not be allowed if we don't pass
the entire request body